src/sql/base/browser/ui/table/media/table.css

```diff
@@ -176,9 +176,8 @@
 	align-items: center;
 }
 
-.slick-icon-cell,
```

Comment:
this is something i forgot to clean up last time. we don't need this class name anymore.